crates/client/src/client.rs

```diff
@@ -1,12 +1,22 @@
-use crate::client::schema::{
-    block::BlockByHeightArgs,
-    coins::{
-        ExcludeInput,
-        SpendQueryElementInput,
+use crate::client::{
+    schema::{
+        block::BlockByHeightArgs,
+        coins::{
+            ExcludeInput,
+            SpendQueryElementInput,
+        },
+        contract::ContractBalanceQueryArgs,
+        tx::DryRunArg,
+        Tai64Timestamp,
+        TransactionId,
+    },
+    types::scalars::{
```

Comment:
We should probably rename this module, as the notion of "scalars" is from GraphQL. However none of these are GraphQL types.

Reply:
Something like `primitives`=)